core-bundle/src/Resources/contao/library/Contao/StringUtil.php

```diff
@@ -1159,7 +1160,7 @@ public static function stripRootDir($path)
 		$rootDir = System::getContainer()->getParameter('kernel.project_dir');
 		$length = \strlen($rootDir);
```

Comment:
We should remove the `$length` variable and use `\strlen($rootDir)` in line 1168.

Reply:
Done in cede8c8bf71643244f07f0325b8d8af553eabf5f.